### ISSUE TITLE
Transfer workspace service settings when overwriting existing GeoServer workspace

### ISSUE THREAD

**Issue by kannes** [GeoCat/qgis-bridge-plugin#176]
**Describe the bug**
Similar to #168 and #169 there are also workspace-specific settings for OGC services (WMS, WFS, WMTS and WCS) possible.

![image](https://github.com/GeoCat/qgis-bridge-plugin/assets/7661092/e88ce982-a9f0-488b-af8e-e1c51f07127d)

Currently, if a workspace has service settings they are dropped when re-creating the workspace.

The plugin should try to preserve them.

**Additional context**
Sorry for dropping these step by step, I am learning GeoServer's funkiness as I go :} Please do in no way feel obligated to respond quickly!

I got this working already and will make another PR.

**Comment by kannes** [comment]
I will recreate https://github.com/kannes/qgis-bridge-plugin/pull/2 here once #169 is merged

**Comment by GeoSander** [comment]
Similarly to #177, please feel free to extract these details from the existing workspace and send them along when you recreate. No one ever asked for it, but it could indeed be a nice addition. Thanks!
